Index: webapp/src/edu/cornell/mannlib/vitro/webapp/search/controller/PagedSearchController.java
===================================================================
--- webapp/src/edu/cornell/mannlib/vitro/webapp/search/controller/PagedSearchController.java	(revision 7634)
+++ webapp/src/edu/cornell/mannlib/vitro/webapp/search/controller/PagedSearchController.java	(working copy)
@@ -896,9 +896,9 @@
         
         //setup HTML format
         resultsToTemplates.put(Result.PAGED, "search-pagedResults.ftl");
-        resultsToTemplates.put(Result.FORM, "search-form.ftl");
-        resultsToTemplates.put(Result.ERROR, "search-error.ftl");
-        resultsToTemplates.put(Result.BAD_QUERY, "search-badQuery.ftl");        
+        resultsToTemplates.put(Result.FORM, "search-pagedResults.ftl");
+        resultsToTemplates.put(Result.ERROR, "search-pagedResults.ftl");
+        resultsToTemplates.put(Result.BAD_QUERY, "search-pagedResults.ftl");        
         templateTable.put(Format.HTML, Collections.unmodifiableMap(resultsToTemplates));
         
         //setup XML format
